buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json

```diff
@@ -476,7 +476,7 @@
       "value": "microsoft/dotnet-buildtools-prereqs"
     },
     "DockerTag": {
-      "value": "ubuntu-14.04-cross-0cd4667-20170319080304",
+      "value": "ubuntu-14.04-cross-e435274-20180426002420",
```

Comment:
I think you should update https://github.com/dotnet/coreclr/blob/0894a0d668634741dc9e73db24b54cc83b556186/buildpipeline/pipelines.json#L134 instead

Reply:
The value in `buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json` is default value which is overridden by `coreclr/buildpipeline/pipelines.json`

Reply:
Thanks @echesakovMSFT, that probably explains why the two values were not the same.  I've reverted the change to DotNet-CoreClr-Trusted-Linux-Crossbuild.json and made the change in pipelines.json.